# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/greetd/bash.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- greetd-0.9.0/greetd/src/session/worker.rs.vanilla	2024-01-06 13:15:12.308047581 +0100
+++ greetd-0.9.0/greetd/src/session/worker.rs	2024-01-06 13:15:29.801472598 +0100
@@ -237,7 +237,7 @@
             }
 
             // Run
-            let cpath = CString::new("/bin/sh").unwrap();
+            let cpath = CString::new("/bin/bash").unwrap();
             execve(
                 &cpath,
                 &[
